.github/workflows/ci.yml

```diff
@@ -52,7 +56,16 @@ jobs:
         continue-on-error: ${{ matrix.development }}
       - name: Check reproducible builds
         run: taskset 1 make check_reproducible
-        if: matrix.otp_release == 'OTP-24.0'
+        if: ${{ matrix.otp_latest }}
+      - name: Build docs
+        if: ${{ matrix.otp_latest }}
+        run: |
+          cd ..
+          git clone https://github.com/elixir-lang/ex_doc.git --branch main --depth 1
+          cd ex_doc
+          ../elixir/bin/mix do local.rebar --force, local.hex --force, deps.get, compile
```

Comment:
```suggestion
          ../elixir/bin/mix do local.rebar --force + local.hex --force + deps.get + compile
```

Reply:
@josevalim I know this has been recently introduced in Elixir. So are we deprecating the usage of comma as separator?

Reply:
In a far away future yes.

Reply:
Sorry, what I meant to ask was, are we replacing commas with plus symbols in Elixir?

Reply:
No. This is unrelated to the language syntax. :)

Reply:
Sorry, this is what I mean.

```
$ ag "mix\s+do\s+[^\n]+, " -U --ignore doc
README.md
192:cd ex_doc && ../elixir/bin/mix do deps.get, compile

lib/mix/lib/mix/tasks/do.ex
39:      mix do compile --list, deps

lib/mix/lib/mix/tasks/archive.install.ex
13:      mix do archive.build, archive.install

lib/mix/lib/mix/tasks/escript.install.ex
13:      mix do escript.build, escript.install

lib/mix/lib/mix/tasks/test.ex
187:          MIX_ENV=test mix do compile --warnings-as-errors, test --warnings-as-errors

```

should these matches be replaced with `+` ?

Reply:
Yes!